Example/healthvault-ios-sdk/MHVFeaturesConfiguration.m

```diff
@@ -24,7 +24,7 @@ + (MHVConfiguration *)configuration
 {
     MHVConfiguration *config = [MHVConfiguration new];
 
-    config.masterApplicationId = [[NSUUID alloc] initWithUUIDString:@"7c81b435-f091-4444-9534-59eb53da66c9"];
+    config.masterApplicationId = [[NSUUID alloc] initWithUUIDString:@"d6318dff-5352-4a10-a140-6c82c6536a3b"];
```

Comment:
Shouldn't check this in, hopefully we can add the record operations to the sample's ID